code/obj/machinery/plantpot.dm

```diff
@@ -1647,7 +1646,7 @@ proc/HYPgeneticanalysis(var/mob/user as mob,var/obj/scanned,var/datum/plant/P,va
 	boutput(user, message)
 	return
 
-proc/HYPnewmutationcheck(var/datum/plant/P,var/datum/plantgenes/DNA,var/obj/machinery/plantpot/PP)
+proc/HYPnewmutationcheck(var/datum/plant/P,var/datum/plantgenes/DNA,var/obj/machinery/plantpot/PP, var/frequencymult = 1)
```

Comment:
never called with a non-1 mult?

Reply:
leftover from adding similar changes to mutation odds that i reverted, though I didn't take it out due to the similarity of the two functions. it's fine reverted or not, though i think it's reasonably likely to be used if anyone actually pokes at botany any time soon.